doc/Options/z390_filetypes.md

```diff
@@ -0,0 +1,32 @@
+PRN|A|ASCII|Output assembly language listing from az390|Text file with the following listing fields:<br/>1) hex relative offset to instruction<br/>2) hex bytes in instruction or data with up ot 8 bytes per line.  Direction PRINT DATA will force listing all data 8 bytes per line.<br/>3) (file#/file-line#)generated-line# identified the file # and the relative file line #for the MLC, MAC, or CPY file that the generated line came from followed by the sequential generated line #.<br/>4) Flag field has + for macro generated lines, and = for line from copybook.<br/>5)BAL line with label, opcode, and operands.
+STA|MALE|ASCII|Output statistics file listing options set, program statistics, and file references|Text file with detail statistics on each step including all option settings, complete macro and copybook cross reference for mz390, and execution statistics.
+TRA|A|ASCII|Trace output file for assembler (lists each line being assembled during each pass)|Text file with ma390 assembler trace output including source file name, relative line, generated line, relative hex offset, and source statement for each assembler pass which may be 2 or more depending on usage of LOCTR.
+TRE|E|ASCII|Trace output file for emulator (lists each instruction, address, and operand values)|Text file with ez390 emulator execution trace output showing, hex psw address, condition code, hex instruction, mnemonic, operand register and storage values at the start of each instruction.  Blank line inserted for each branch to non sequential address.  The high bit of PSW hex address indicates 24 or 31 bit address mode.
+TRL|L|ASCII|Trace output file for linker (lists each OBJ record processed)|Text file with lz390 linker trace output including ESD external symbol hex addresses and lengths, TXT text records with relative offset within ESD and hex data.  RLD relocation records showing hex offset, field length, and ESD symbol.
+TRM|M|ASCII|Trace output file for macro process (TRACEM lists each macro line executed, TRACEP lists every macro pseudo code instruction executed showing the current value of local and global variables used in SET statements and AIF statements etc.)|Text file with mz390 macro processor trace output file which is typically generated in parallel with the generation of the first pass information in the TRA file since mz390 and az390 run as concurrent tasks.  This file includes source file name, relative line number, and source macro statement.  For generated BAL output lines the source file# and relative line# cross reference is also include.  If the TRACEP option is specified, this file also includes listing of the generation and execution of macro pseudo code instructions.
```

Comment:
On 2nd line for line 29. "... statements etc.)" -> "... statements etc).".

Reply:
Resolved by 52df1cf